{"uuid": "ea6f087f-8422-41af-82d3-15d4ae42dbbf", "children": ["cb6674d4-12df-48bc-a6b2-a6b59f48ce8f"], "befores": [{"name": "browser", "status": "passed", "start": 1683537086419, "stop": 1683537087498}], "afters": [{"name": "browser::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\allure_commons\\_allure.py\", line 200, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 899, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\user\\PycharmProjects\\pythonProject1\\conftest.py\", line 18, in browser\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 238, in quit\n    self.service.stop()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 146, in stop\n    self.send_remote_shutdown_command()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 122, in send_remote_shutdown_command\n    request.urlopen(\"%s/shutdown\" % self.service_url)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 222, in urlopen\n    return opener.open(url, data, timeout)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 525, in open\n    response = self._open(req, data)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 542, in _open\n    result = self._call_chain(self.handle_open, protocol, protocol +\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 502, in _call_chain\n    result = func(*args)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 1383, in http_open\n    return self.do_open(http.client.HTTPConnection, req)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\urllib\\request.py\", line 1354, in do_open\n    h.request(req.get_method(), req.selector, req.data, headers,\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 1252, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 1298, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 1247, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 1007, in _send_output\n    self.send(msg)\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 947, in send\n    self.connect()\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\http\\client.py\", line 918, in connect\n    self.sock = self._create_connection(\n  File \"c:\\users\\user\\appdata\\local\\programs\\python\\python38\\lib\\socket.py\", line 796, in create_connection\n    sock.connect(sa)\n"}, "start": 1683537143312, "stop": 1683537147489}], "start": 1683537086419, "stop": 1683537147489}